3-sqlite-server/3-2-alter-tasks/server.js

```diff
@@ -0,0 +1,51 @@
+import Database from "better-sqlite3";
+import * as queries from "./queries.js";
+import fs from "fs";
+
+const dbPath = "./test.db";
+
+if (fs.existsSync(dbPath)) {
+  console.log("test.dbが存在するので削除します...\n");
+
+  fs.unlinkSync(dbPath);
+}
+
+const db = new Database("test.db");
+
+const displayAllTasks = (db, title) => {
+  const rows = db.prepare(queries.Tasks.readAll).all();
+  console.log(`---- ${title} ----`);
+  rows.forEach((row) => {
+    console.log(`${row.id}: ${row.title} (completed: ${row.completed})`);
+  });
+  console.log();
+};
```

Comment:
そもそもこのファイルのDB宣言の下にこの関数定義するなら引数にdbいらなくない？
```suggestion
const displayAllTasks = (title) => {
  const rows = db.prepare(queries.Tasks.readAll).all();
  console.log(`---- ${title} ----`);
  rows.forEach((row) => {
    console.log(`${row.id}: ${row.title} (completed: ${row.completed})`);
  });
  console.log();
};
```